sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -4600,3 +4600,118 @@ case class ArrayExcept(left: Expression, right: Expression) extends ArrayBinaryL
   override protected def withNewChildrenInternal(
     newLeft: Expression, newRight: Expression): ArrayExcept = copy(left = newLeft, right = newRight)
 }
+
+/**
+ * Given an array, and another element append the element at the end of the array.
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(array, element) - Append the element",
+  examples =
+    """
+    Examples:
+      > SELECT _FUNC_(array('b', 'd', 'c', 'a'), 'd');
+
+  """,
+  since = "3.4.0",
+  group = "array_funcs")
+case class ArrayAppend(left: Expression, right: Expression)
+  extends BinaryExpression
+    with ImplicitCastInputTypes with QueryErrorsBase {
```

Comment:
nit: 2 space

Reply:
Didn't understand this one! Can you please elaborate a little on this one. Thanks!

Reply:
Ahh got it ! Thanks ! Let me fix this! Do you use any specific plugin with IDE ? That highlights all these, I have added checkstyle extension but it didn't highlight such things. :)

Reply:
Unfortunately, I don't have a plug-in can highlights this ...

Reply:
Got it !